playbooks/common/openshift-cluster/upgrades/docker/restart.yml

```diff
@@ -19,11 +19,9 @@
   when: openshift.common.is_containerized | bool
 
 - name: Wait for master API to come back online
-  become: no
-  local_action:
-    module: wait_for
-      host="{{ inventory_hostname }}"
-      state=started
-      delay=10
-      port="{{ openshift.master.api_port }}"
-  when: inventory_hostname in groups.oo_masters_to_config
+  wait_for:
+    host: "{{ openshift.common.hostname }}"
+    state: started
+    delay: 10
+    port: "{{ openshift.master.api_port }}"
+  when: openshift.common.hostname in groups.oo_masters_to_config
```

Comment:
ok my test is about to pass because it just skips the api waiting, I will re-try with that fix.